yarn.lock

```diff
@@ -8563,7 +8563,7 @@ embark-i18n@^5.2.3:
     i18n "0.8.3"
     os-locale "4.0.0"
 
-embark-logger@^5.1.1-nightly.2, embark-logger@^5.2.3:
+embark-logger@^5.2.3:
```

Comment:
Whenever `embark[js][-*]` shows up in `yarn.lock` it means that some caret range needs to be adjusted in dev/deps and/or a `"version"` field may need to be adjusted.

Also, I'm not seeing typechecking related errors on `master`; are these changes still needed for other reasons?